FILTERS.md

```diff
@@ -1,22 +1,31 @@
 # Filters
 
-When a new `Processor` is requested using the endpoint `/api/v1/bridges/{id}/processors` it is possible to specify one or more filters to apply to the ingress event. If there is a match, then the `action` of the processor is called. 
+When a new `Processor` is requested using the endpoint `/api/v1/bridges/{id}/processors` it is possible to specify one or more `Filters` to apply to `Events` sent to your Bridge instance.
+If the `Filter` you provide matches an `Event` then the [Transformation](TRANSFORMATIONS.md) and [Action](ACTIONS.md) of the associated `Processor` are invoked. 
 
-Every filter has 3 properties to specify: 
+If you do not specify a `Filter` definition for your `Processor`, then your `Processor` will match all `Events` sent to your Bridge Instance.
 
-- `type`: the type of the filter. This can be `StringEquals`, `StringBeginsWith`, `StringContains`.
-- `key`: The field in the event data that you're using for filtering. It can be a number, boolean, string. **Arrays are not supported yet**.
-- `values`: The value or values to compare to the key.
+If there are multiple `Filters` defined for a `Processor`, then all `Filters` must match the `Event` for the [Transformation](TRANSFORMATIONS.md) and [Action](ACTIONS.md) to be invoked.  
 
-For events in **Cloud Events schema**, you can use the values for the key: `eventid`, `source`, `eventtype`, `eventtypeversion`, or event data (like `data.key1`). You can navigate the object with the `.` (dot) notation.
+## Properties of a Filter
 
-## Filter types
+Every `Filter` has 3 properties to specify: 
 
-The available filter types are: 
+- `type`: the type of the `Filter`. This must be one of the supported `Filter` types listed below
+  - Attempting to use an unknown `Filter` type will result in an Error from the Bridge API.
+- `key`: The field in the `Event` that you want to filter on. 
+  - This must be a single field only. Arrays of fields to match on are not yet supported. 
+- `values`: The value or values to compare to the field identified by the key.
```

Comment:
in some filters we have `value` and in some others `values`. For example for `StringEquals` we have `value` and for `StringIn` we have `values`